packages/server/tests/helpers/utils.ts

```diff
@@ -372,4 +383,229 @@ export class Utils {
   static async wait(time: number): Promise<void> {
     await new Promise((r) => setTimeout(r, time));
   }
+
+  static async writeHeapSnapshotAsync(): Promise<string | undefined> {
+    return new Promise((resolve, reject) => {
+      writeSnapshot((error, fileName) => {
```

Comment:
qq: what is fileName? Is it something predefined within writeSnapshot() method?

Reply:
Yeah, it’s an automated file name `heapdump-<timestamp>.json`